### PR TITLE
Fixing insights label styles

### PR DESCRIPTION
**Pull request by Fewwy**
The labels had gray borders that were incorrect (the PatternFly styling guide says that they need to be colored)
https://www.patternfly.org/v4/components/label/html
I removed unnecessary code that was adding custom classes and used the native PatternFly class to fix the CSS issue.
I left the CSS code for 1 class that covers the "gold" color because it is not provided by PatternFly labels color selection.
image

### CODE REVIEW

**Comment by Fewwy** [comment]
@gkarat @Hyperkid123 PTAL

File: packages/components/src/InsightsLabel/InsightsLabel.tsx [hunk 1]
```diff
@@ -34,7 +33,14 @@ export interface InsightsLabelProps extends LabelProps {
 
 const InsightsLabel: React.FunctionComponent<InsightsLabelProps> = ({ value = 1, text, hideIcon, className, rest, ...props }) => {
   return (
-    <Label {...rest} {...props} className={classNames(className, `ins-c-label-${value}`)} icon={!hideIcon && VALUE_TO_STATE[value].icon}>
+    <Label
+      {...rest}
+      {...props}
+      className={
+        value === 4 ? 'pf-c-label pf-m-red' : value === 3 ? 'pf-c-label pf-m-orange' : value === 1 ? 'pf-c-label pf-m-blue' : 'ins-c-label-2'
```
Comment (gkarat) [inline packages/components/src/InsightsLabel/InsightsLabel.tsx:40]: `pf-c-label` can be omitted here I believe.

Also, consider using an object to map total risk value to the appropriate class name like:
```
{ 1: 'pf-m-blue', 2: 'ins-c-label-2', ... }
```

File: packages/components/src/InsightsLabel/labels.scss [hunk 2]
```diff
@@ -42,13 +13,6 @@
...
-    background-color: var(--ins-c-label--blue-bg);
-    .pf-c-label__content {
-        color: var(--inc-c-label--content-blue);
-        .pf-c-label__icon {
-            color: var(--inc-c-label--icon-blue);
-        }
-    }
-}
+.ins-c-label-2.pf-c-label {.pf-c-label__content::before {
+    border: 1px solid var(--inc-c-label--icon-gold);
```
Comment (gkarat) [inline packages/components/src/InsightsLabel/labels.scss:17]: I have a feeling that the colors from https://marvelapp.com/prototype/g70a1he/screen/84434999 are different. Could you please double check it?

File: packages/components/src/InsightsLabel/InsightsLabel.tsx (hunk 1 above)
Comment (rvsia) [inline packages/components/src/InsightsLabel/InsightsLabel.tsx:40]: Could we add that to `VALUE_TO_STATE` to remove these ternary operators?

File: packages/components/src/InsightsLabel/labels.scss (hunk 2 above)
Comment (Fewwy) [reply packages/components/src/InsightsLabel/labels.scss:17]: Good catch. I asked Marie to give me the exact # number of the color.

File: packages/components/src/InsightsLabel/InsightsLabel.tsx (hunk 1 above)
Comment (Fewwy) [reply packages/components/src/InsightsLabel/InsightsLabel.tsx:40]: @rvsia Good suggestion. I will do that

File: packages/components/src/InsightsLabel/labels.scss (hunk 2 above)
Comment (Fewwy) [reply packages/components/src/InsightsLabel/labels.scss]: Updated the colors according to the Marie's mockup

**Review by rvsia — APPROVED** [review]

File: packages/components/src/InsightsLabel/InsightsLabel.tsx [hunk 3]
```diff
@@ -6,19 +6,19 @@ import { CriticalRiskIcon } from '@patternfly/react-icons';
...
     text: string;
+    class: string;
   };
 };
 const VALUE_TO_STATE: InsightsLabelValueMapping = {
-  1: { icon: <AngleDoubleDownIcon />, text: 'Low' },
-  2: { icon: <EqualsIcon />, text: 'Moderate' },
-  3: { icon: <AngleDoubleUpIcon />, text: 'Important' },
-  4: { icon: <CriticalRiskIcon />, text: 'Critical' },
+  1: { icon: <AngleDoubleDownIcon />, text: 'Low', class: 'pf-c-label pf-m-blue' },
```
Comment (gkarat) [inline packages/components/src/InsightsLabel/InsightsLabel.tsx:18]: No reason to add `pf-c-label` here. The `Label` component already adds it while rendered.

File: packages/components/src/InsightsLabel/labels.scss [hunk 4]
```diff
@@ -1,36 +1,8 @@
...
-
-.ins-c-label-3.pf-c-label {
-    background-color: var(--ins-c-label--orange-bg);
-    .pf-c-label__content {
-        color: var(--inc-c-label--content-orange);
-        .pf-c-label__icon {
-            color: var(--inc-c-label--icon-orange);
-        }
-    }
+    --inc-c-label--border-gold:         #F9E0A2;
```
Comment (gkarat) [inline packages/components/src/InsightsLabel/labels.scss:5]: Nit-pick: you can refer to `--pf-global--palette--gold-100` which has the same value: https://www.patternfly.org/v4/guidelines/colors/#color-palette

**Review by gkarat — APPROVED** [review]
LGTM. A couple of things can be even more improved, but it's not crucial

File: packages/components/src/InsightsLabel/InsightsLabel.tsx [hunk 5]
```diff
@@ -6,19 +6,19 @@ import { CriticalRiskIcon } from '@patternfly/react-icons';
...
+  2: { icon: <EqualsIcon />, text: 'Moderate', class: 'ins-c-label-2' },
+  3: { icon: <AngleDoubleUpIcon />, text: 'Important', class: 'pf-c-label pf-m-orange' },
+  4: { icon: <CriticalRiskIcon />, text: 'Critical', class: 'pf-c-label pf-m-red' },
```
Comment (Hyperkid123) [inline packages/components/src/InsightsLabel/InsightsLabel.tsx:21]: Do now use class. Use the LabelComponent props API instead. For the colors it should be the `color` prop. https://www.patternfly.org/v4/components/label-group/#label.

File: packages/components/src/InsightsLabel/InsightsLabel.tsx [hunk 6]
```diff
@@ -34,7 +34,7 @@ export interface InsightsLabelProps extends LabelProps {
 
 const InsightsLabel: React.FunctionComponent<InsightsLabelProps> = ({ value = 1, text, hideIcon, className, rest, ...props }) => {
   return (
-    <Label {...rest} {...props} className={classNames(className, `ins-c-label-${value}`)} icon={!hideIcon && VALUE_TO_STATE[value].icon}>
+    <Label {...rest} {...props} className={VALUE_TO_STATE[value].class} icon={!hideIcon && VALUE_TO_STATE[value].icon}>
```
Comment (Hyperkid123) [inline packages/components/src/InsightsLabel/InsightsLabel.tsx:37]: ^^

File: packages/components/src/InsightsLabel/InsightsLabel.tsx (hunk 5 above)
Comment (Fewwy) [reply packages/components/src/InsightsLabel/InsightsLabel.tsx:21]: Ok, sure

File: packages/components/src/InsightsLabel/InsightsLabel.tsx (hunk 3 above)
Comment (Fewwy) [reply packages/components/src/InsightsLabel/InsightsLabel.tsx]: Yep, you are correct

File: packages/components/src/InsightsLabel/labels.scss (hunk 4 above)
Comment (Fewwy) [reply packages/components/src/InsightsLabel/labels.scss]: Yep, changed to the var

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**